src/pages/MyProfile/MyProfile.js

```diff
@@ -71,7 +71,7 @@ const MyProfile = ({ history: { push } }) => {
           You need to set your name.
         </small>}
       </label>
-      <Button variant='link' onClick={showTos}>
+      <Button variant='link' onClick={showTos} dataTestId='tos-button'>
```

Comment:
can you remove `data-testid` here?

And use `getByText('View Terms of Service')` in the test.

Reply:
I was using the testId here as there actually are two instances of 'View Terms of Service', one in the nav bar, and one on the profile page itself.  Nonetheless, I can preempt which order they will arrive and can use `getAllByText`, if you think that is the best approach.